### DISCUSSION TITLE
Simpler License Terms — ideally MIT/BSD compatible

### DISCUSSION THREAD

**Discussion by jaukia** [Myndex/apca-w3#14] [General]
Looking at the current license terms, it is very difficult for me to understand what the implications of using apca-w3 in my web app would be and what risks are related to it. I think in both web app companies and OS community, it is often the case that only MIT/BSD compatible code is approved and tools with any custom licenses are avoided, since they would require an (expensive) legal check and potentially cause risks for company financing etc.

I can work around that and even consider using WCAG 2.0 contrast if I have to. But I'm worried that this will slow down adoption of the better contrast calculations, since many open-source and commercial tool devs might be hesitant to take the library in the use. 

It is your right to decide what license to use, but just wanted to share this insight — though perhaps this is obvious for you already.

https://github.com/Myndex/apca-w3/blob/master/LICENSE.md

**Comment by TheJaredWilcurt** [comment]
Basically everything in the JS ecosystem is MIT licensed. It's something like 70+% of npm packages from the last 5 years use MIT. Though now I can't find the source of that (just tried). But there are plenty of sources showing MIT as used by the majority of JS libraries. It's what the community is comfortable and familiar with.

Not saying this library needs to be MIT, but it would certainly be the smoothest path for adoption if it's an option.
